pycbc/io/hdf.py

```diff
@@ -907,14 +907,16 @@ def to_coinc_xml_object(self, file_name):
 
             # Set up sngls
             # FIXME: As two-ifo is hardcoded loop over all ifos
-            sngl_combined_mchirp = 0
-            sngl_combined_mtot = 0
+            sngl_mchirps = []
+            sngl_mtots = []
             net_snrsq = 0
+            involved_ifos = []
```

Comment:
to be clear, I would rather call this 'triggered_ifos'

Reply:
that makes more sense